applications/FluidDynamicsApplication/custom_elements/surface_smoothing_element.h

```diff
@@ -0,0 +1,375 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Mohammad R. Hashemi
+//
+
+#if !defined(KRATOS_SURFACE_SMOOTHING_ELEMENT_H_INCLUDED )
+#define KRATOS_SURFACE_SMOOTHING_ELEMENT_H_INCLUDED
+
+// System includes
+#include <string>
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "includes/element.h"
+#include "utilities/element_size_calculator.h"
+#include "includes/deprecated_variables.h" //For IS_STRUCTURED
+
+// Include Base headers
+#include "includes/checks.h"
+
+// Application includes
+#include "fluid_dynamics_application_variables.h"
```

Comment:
please move includes to the cpp if you are not using them in the header

Reply:
Done.